README.md

```diff
@@ -2,16 +2,35 @@ The web-platform-tests Project [![IRC chat](https://goo.gl/6nCIks)](http://irc.w
 ==============================
 
 The web-platform-tests Project is a W3C-coordinated attempt to build a
-cross-browser testsuite for the Web-platform stack. Writing tests in a way
+cross-browser test suite for the Web-platform stack. Writing tests in a way
 that allows them to be run in all browsers gives browser projects
 confidence that they are shipping software that is compatible with other
-implementations, and that later implementations will be compatible with
-their implementations. This in turn gives Web authors/developers
+implementations and those later implementations will be compatible with
+their implementations. This, in turn, gives Web authors/developers
 confidence that they can actually rely on the Web platform to deliver on
 the promise of working across browsers and devices without needing extra
-layers of abstraction to paper over the gaps left by specification
+layers of abstraction to paper over the gaps left by the specification
```

Comment:
We're talking about specifications generally here, so there should definitely be no "the" here.